dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
-        "label": "HUNTER HOLMES MCGUIRE 2 MOB CL"
+        "label": "Hunter Holmes McGuire 2 VA Mobile Clinic"
       },
       {
         "value": "652GG",
-        "label": "RICHMOND VA MOBILE CLINIC"
+        "label": "Richmond 1 VA Mobile Clinic"
       },
       {
         "value": "590GD",
-        "label": "CHESAPEAKE VA CLINIC"
+        "label": "Chesapeake VA Clinic"
       },
       {
         "value": "652GB",
-        "label": "FREDERICKSBURG 2 VA CLINIC"
+        "label": "Fredericksburg 2 VA Clinic"
       },
       {
         "value": "590",
-        "label": "HAMPTON VA MEDICAL CENTER"
+        "label": "Hampton VA Medical Center"
       },
       {
         "value": "652",
-        "label": "HUNTER HOLMES MCGUIRE HOSPITAL"
+        "label": "Hunter Holmes McGuire Hospital"
       },
       {
         "value": "658",
-        "label": "SALEM VA MEDICAL CENTER"
+        "label": "Salem VA Medical Center"
       }
     ],
     "AR": [
       {
         "value": "614GB",
-        "label": "JONESBORO VA CLINIC"
+        "label": "Jonesboro VA Clinic"
       },
       {
         "value": "564GA",
-        "label": "HARRISON VA CLINIC"
+        "label": "Harrison VA Clinic"
       },
       {
         "value": "564GB",
-        "label": "FORT SMITH VA CLINIC"
+        "label": "Fort Smith VA Clinic"
       },
       {
         "value": "598GD",
-        "label": "MENA, AR (CBOC)"
+        "label": "Mena VA Clinic"
       },
       {
         "value": "598A0",
-        "label": "NORTH LITTLE ROCK, AR VANPH"
+        "label": "Eugene J. Towbin Healthcare Center"
       },
       {
         "value": "598GA",
-        "label": "MOUNTAIN HOME, AR CBOC"
+        "label": "Mountain Home VA Clinic"
       },
       {
         "value": "598GB",
-        "label": "EL DORADO, AR CBOC"
+        "label": "El Dorado VA Clinic"
       },
       {
         "value": "598GC",
-        "label": "HOT SPRINGS, AR CBOC"
+        "label": "Hot Springs VA Clinic"
       },
       {
         "value": "598GE",
-        "label": "PINE BLUFF CBOC"
+        "label": "Pine Bluff VA Clinic"
       },
       {
         "value": "598GF",
-        "label": "SEARCY CBOC"
+        "label": "Searcy VA Clinic"
       },
       {
         "value": "667GA",
-        "label": "TEXARKANA CBOC"
+        "label": "Texarkana VA Clinic"
       },
       {
         "value": "614GN",
-        "label": "HELENA VA CLINIC"
+        "label": "Helena VA Clinic"
       },
       {
         "value": "564GD",
-        "label": "OZARK CBOC"
+        "label": "Ozark VA Clinic"
       },
       {
         "value": "598GG",
-        "label": "CONWAY CBOC"
+        "label": "Conway VA Clinic"
       },
       {
         "value": "598GH",
-        "label": "RUSSELLVILLE CBOC"
+        "label": "Russellville VA Clinic"
       },
       {
         "value": "598",
-        "label": "CENTRAL ARKANSAS HEALTH CARE SYSTEM - LITTLE ROCK"
+        "label": "John L. McClellan Memorial Veterans' Hospital"
```

Comment:
Let's confirm this name change.

Reply:
Looks good.

**References**: 
https://www.va.gov/find-locations/facility/vha_598
https://www.littlerock.va.gov/locations/directions.asp